ui/selectors/multichain-accounts/account-tree.ts

```diff
@@ -688,3 +688,36 @@ export const getNetworkAddressCount = createSelector(
     return wallet.groups[accountGroupId].accounts.length;
   },
 );
+
+/**
+ * Get all internal accounts from a specific account group by its ID.
+ *
+ * @param state - The Redux state.
+ * @param groupId - The ID of the account group.
+ * @returns Array of internal accounts in the specified group, or empty array if not found.
+ */
+export const getInternalAccountsFromGroupById = createDeepEqualSelector(
```

Comment:
TODO: Getting this warning from tests will need to fix
```
  The result function returned its own inputs without modification. e.g
    `createSelector([state => state.todos], todos => todos)`
    This could lead to inefficient memoization and unnecessary re-renders.
```